File: iniciante/c99/1041/1041.c
#include <stdio.h>

int main() {

	double x, y;

	scanf("%lf", &x);
	scanf("%lf", &y);

	if (x > 0) {
		if (y > 0) 
			printf("Q1\n");

		else {
			if (y < 0) 
				printf("Q4\n");

			else 
				printf("Eixo X\n");
		}
	}

	else if (x < 0) {
		if (y < 0) 
			printf("Q3\n");
	
		else {
			if(y > 0)
				printf("Q2\n");
			
			else
				printf("Eixo X\n");

		}
	}

	else {
		if (x == 0 && y != 0)
			printf("Eixo Y\n");

		else
			printf("Origem\n");
	}
	
	return 0;

}

File: python/1041.py
#1041 - Coordenadas de um Ponto

coords = input().split()

x = float(coords[0])
y = float(coords[1])

if (x > 0 or x < 0) and (y > 0 or y < 0):
    if x > 0 and y > 0:
        print("Q1")
    elif x < 0 and y > 0:
        print("Q2")
    elif x < 0 and y < 0:
        print("Q3")
    else:
        print("Q4")
    
else:
    if x == 0 and y != 0:
        print("Eixo Y")
    elif x != 0 and y == 0:
        print("Eixo X")
    else:
        print("Origem")